pygmt/figure.py

```diff
@@ -167,8 +172,24 @@ def psconvert(self, **kwargs):
             using the input names as base, which are appended with an
             appropriate extension. Use this option to provide a different name,
             but without extension. Extension is still determined automatically.
-        icc_gray : bool
-            Enforce gray-shades by using ICC profiles.
+        icc_gray : str
```

Comment:
> > > I'm not fully sure which parts need to be changed here @GenericMappingTools/pygmt-maintainers . Every change that was introduced in [GenericMappingTools/gmt#5583](https://github.com/GenericMappingTools/gmt/pull/5583)?
> > 
> > I would also update `-I` (icc_gray) as a minimum since that is aliased. The changes in `-N` and `-W` could be done in a separate PR when we add long aliases for those.
> 
> I agree, however, it seems like ``icc_gray`` isn't a good alias for ``-I`` anymore @weiji14.

Yeah, this seems completely different. Should we rename it to margin, scale, or something else? This would need to require a deprecation cycle too 😅

Reply:
> Should we rename it to margin, scale, or something else?

Is `resize` better?

Reply:
Would also go with `resize`. What do @GenericMappingTools/pygmt-maintainers mean?

Reply:
Not perfect, but I think `resize` will do. Unless someone can come up with a better name 🙃

Reply:
`resize` works for me

Reply:
Ok, go let's go with `resize` then @michaelgrund:

```suggestion
        resize : str
```

Also, please modify:
- L132: `I="icc_gray",` -> `I="resize",`
- L127, add `@deprecate_parameter("icc_gray", "resize", "v0.6.0", remove_version="v0.8.0")` (below `@fmt_docstring`)

Make sure to change the PR title to ''Figure.psconvert: Deprecate parameter "icc_gray" to "resize" (remove in v0.8.0)" and update the PR label to 'deprecation' so that it goes in the changelog. Follow steps detailed at https://www.pygmt.org/v0.5.0/maintenance.html#backwards-compatibility-and-deprecation-policy.